app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/preferenceCommunicatorFactory.ts

```diff
@@ -0,0 +1,17 @@
+const preferenceCommunicatorFactory = (window) => ({
+  sendPreferences: (userPreferences) => {
```

Comment:
Could this not just take the actual preferences rather than the userPreferences object? Worth adding a type too.